proto/encore/parser/meta/v1/meta.proto

```diff
@@ -290,6 +290,7 @@ message PathSegment {
     PARAM = 1;
     WILDCARD = 2;
     FALLBACK = 3;
+    ROOT = 4;
```

Comment:
Thoughts on representing this instead as a LITERAL segment with value `""`?

Reply:
would be nice not to have to update the protos, I just added ROOT so its more explicit, but I'm happy to change to that!

Reply:
I think I'd rather do the LITERAL; it seems the logic is the same for literals everywhere. And otherwise we need to modify a lot of other code (dev dashboard, etc) to handle the new proto enum case